src/main/scala/dpla/ingestion3/model/DplaMapData.scala

```diff
@@ -137,7 +140,19 @@ case class EdmPlace(
                    )
 
 case class EdmTimeSpan(
-                        originalSourceDate: ZeroToMany[String] = Seq(),
+                        /* Why is this 0.n? I would have assumed ExactlyOne or ZeroToOne
+                            since it wouldn't make sense to have multiple original ranges
+                            per EdmTimeSpan but rather one edmTimeSpan per originalSourceDate?
+
+                            Esp. since in date in sourceResource is also ZeroToMany...
+
+                            I'm changing the type for the enrichment driver and need to follow-up
+                            w/team
```

Comment:
I don't know enough to contribute to this discussion at this point, so you should probably follow up with Mark or Michael.